### PR TITLE
Add golangci-lint config file and fix issues

### LINKED ISSUE #66: Add golanci.yml and fix issues
I've already started working on this.

Partially fixed here https://github.com/bcneng/candebot/pull/67

### PR DESCRIPTION
**Pull request by gonzaloserrano**
Fixes #66

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Tetrate, Inc 2021 All Rights Reserved.
+
```
Comment (ignasi35) [inline .golangci.yml:2]:
```suggestion
```

File: .golangci.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Tetrate, Inc 2021 All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (gonzaloserrano) [reply .golangci.yml:1]: Lol copied the Tetrate one but then removed 75% of it

**Review by ignasi35 — APPROVED** [review]
LGTM

**Review by smoya — APPROVED** [review]
LGTM

**Review by sdecandelario — APPROVED** [review]

**Pull request merged**